gnssanalysis/gn_io/sp3.py

```diff
@@ -317,44 +340,51 @@ def clk_log(x):
     def prn_formatter(x):
         return f"P{x}"
 
+    # TODO NOTE
```

Comment:
No note left here?

Reply:
Ah, I added this line just to more clearly draw attention to the large block of commentary already there.

Specifically the fact we're not precicely following the SP3 standard, as we have whitespace between columns. If we have reason to rework our SP3 writing in future, this feels like an important motivator to include in the discussion.